sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_data_lake_file_client_async.py

```diff
@@ -188,6 +189,66 @@ def __init__(
         blob_properties = await self._get_path_properties(**kwargs)
         return FileProperties._from_blob_properties(blob_properties)  # pylint: disable=protected-access
 
+    async def upload_data(self, data,  # type: Union[AnyStr, Iterable[AnyStr], IO[AnyStr]]
+                    length=None,  # type: Optional[int]
+                    overwrite=None,  # type: Optional[bool]
```

Comment:
The `overwrite` parameter should to default to `False`. What does `None` mean in this case?

Reply:
Good catch! It should be false